Contest_API.md

```diff
@@ -1982,16 +1982,43 @@ The following endpoints are associated with commentary:
 
 Properties of commentary objects:
 
-| Name          | Type          | Description
-| :------------ | :------------ | :----------
-| id            | ID            | Identifier of the commentary.
-| time          | TIME          | Time of the commentary message.
-| contest\_time | RELTIME       | Contest time of the commentary message.
-| message       | string        | Commentary message text. May contain special tags for [teams](#teams) and [problems](#problems) on the format `#t<team ID>` and `#p<problem ID>` respectively.
-| team\_ids     | array of ID ? | JSON array of [team](#teams) IDs the message is related to.
-| problem\_ids  | array of ID ? | JSON array of [problem](#problems) IDs the message is related to.
-
-For the message, if an literal `#` is needed, `\#` must be used. Similarly for literal `\`, `\\` must be used.
+| Name          | Type            | Description
+| :------------ | :-------------- | :----------
+| id            | ID              | Identifier of the commentary.
+| time          | TIME            | Time of the commentary message.
+| contest\_time | RELTIME         | Contest time of the commentary message.
+| message       | string          | Commentary message text. May contain special tags refering to endpoint objects on the format `{<endpoint>:<object ID>}`. E.g. this is most commonly used for references to [teams](#teams) and [problems](#problems) as `{teams:<team ID>}` and `{problems:<problem ID>}` respectively.
```

Comment:
```suggestion
| message       | string          | Commentary message text. May contain special tags referring to endpoint objects in the format `{<endpoint>:<object ID>}`. This is most commonly used for references to [teams](#teams) and [problems](#problems) as `{teams:<team ID>}` and `{problems:<problem ID>}` respectively.
```

Reply:
Huh... is it really "in the format" and not "on the format"?

Reply:
I _think_ so, but I'm not a native speaker, so maybe @deboer-tim should decide :-)

Reply:
Not sure I should be calling the shots given my grammar some days. :) IMHO 'in' is better than 'on', could also use 'with' or 'using'. Maybe 'using' is best?

Reply:
Fixed (with "using").